scripts/common.py

```diff
@@ -150,6 +150,14 @@ def require_cwd_to_be_oppia(allow_deploy_dir=False):
 
 def open_new_tab_in_browser_if_possible(url):
     """Opens the given URL in a new browser tab, if possible."""
+    python_utils.PRINT(
```

Comment:
I suggest that you cache this preference, per run of the script, otherwise it becomes irritating to type an extra "y" each time.

Reply:
Done.

Reply:
AFAIR, this was made so that the links are opened after reading the task need to perform on that page. I don't think caching is required here! Any thoughts? @seanlip @ankita240796

Reply:
Sorry, I missed this comment. So, my thought here is that if a script is calling the `open_new_tab` function multiple times, we should cache since it does not make sense to ask the same question multiple times in a single run.